### PR TITLE
Fixing test failure

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Related PR https://github.com/aspnet/RazorTooling/pull/107
@pranavkm / @NTaylorMullen

### CODE REVIEW

File: src/RazorToolingTestApp.LibraryPackage/project.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0",
   "dependencies": {
-    "Microsoft.AspNetCore.Razor.Runtime": "1.1.0-*",
+    "Microsoft.AspNetCore.Razor.Runtime": "1.0.0",
```
Comment (pranavkm) [inline src/RazorToolingTestApp.LibraryPackage/project.json:4]: Should we have an alternate set of tests that verify this works with the new runtime?

**Review by pranavkm — CHANGES_REQUESTED** [review]

File: src/RazorToolingTestApp.LibraryPackage/project.json (hunk 1 above)
Comment (NTaylorMullen) [reply src/RazorToolingTestApp.LibraryPackage/project.json:4]: Ya we should. That can go in separately though. @ajaybhargavb could you file an issue to create cross-versioned tests?

File: src/Microsoft.AspNetCore.Razor.Tools/project.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
           "type": "platform",
           "version": "1.0.0"
         },
+        "System.Runtime.InteropServices.RuntimeInformation": "4.3.0-*",
```
Comment (NTaylorMullen) [inline src/Microsoft.AspNetCore.Razor.Tools/project.json:44]: When you had your primitives 4.3 was this needed? Just curious so I know where this is coming in from.

**Review by pranavkm — APPROVED** [review]

File: src/Microsoft.AspNetCore.Razor.Tools/project.json (hunk 2 above)
Comment (ajaybhargavb) [reply src/Microsoft.AspNetCore.Razor.Tools/project.json:44]: This is needed either way.

Comment (NTaylorMullen) [reply src/Microsoft.AspNetCore.Razor.Tools/project.json:44]: What requires it?

Comment (ajaybhargavb) [reply src/Microsoft.AspNetCore.Razor.Tools/project.json:44]: https://github.com/aspnet/RazorTooling/blob/dev/src/Microsoft.AspNetCore.Razor.Tools/Internal/RazorProjectLoadContext.cs#L34

File: src/RazorToolingTestApp.LibraryPackage/project.json (hunk 1 above)
Comment (ajaybhargavb) [reply src/RazorToolingTestApp.LibraryPackage/project.json:4]: Filed an issue. https://github.com/aspnet/RazorTooling/issues/109

**Pull request merged**